operations/paymentrequests.md

```diff
@@ -0,0 +1,272 @@
+            "AccountId": "8466dfdd-0964-4002-8719-afa900d0f1ba",
+            "Amount": {
+                "Currency": "EUR",
+                "Value": 10
+            },
+            "Type": "Payment",
+            "Reason": "PaymentCardMissing",
+            "ExpirationUtc": "2023-02-20T12:00:00.000Z",
+            "Description": "Payment required",
+            "Notes": "Internal notes."
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `PaymentRequests` | array of [Payment request parameters](#payment-request-parameters) | required, max 1000 items | Payment requests to be added. |
+
+#### Payment request parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AccountId` | string | required | Unique identifier of the [Customer](customers.md#customer) account to which the payment request is issued. |
+| `Amount` | [Currency value](#currency-value) | required | Amount of the payment request. |
+| `Type` | [Payment request type](#payment-request-type) | required | A payment request type. |
+| `Reason` | [Payment request reason](#payment-request-reason) | required | A payment request reason. |
+| `ExpirationUtc` | string | required | Date and time of the payment request's expiration in ISO 8601 format. |
+| `Description` | string | required, max 1000 characters | Description of the payment request. |
+| `Notes` | string | optional, max 1000 characters | Payment request's notes. |
+
+#### Currency value
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Currency` | string | required | ISO-4217 code of the [Currency](currencies.md#currency). |
+| `Value` | number | required | Amount in the currency. |
+
+### Response
+
+```javascript
+{
+    "PaymentRequests": [
+        {
+            "Id": "6282d17b-a068-4a9f-83d3-afae00c39bfb",
+            "AccountId": "8466dfdd-0964-4002-8719-afa900d0f1ba",
+            "ReservationGroupId": null,
+            "State": "Pending",
+            "Amount": {
+                "Currency": "EUR",
+                "NetValue": 10,
+                "GrossValue": 10,
+                "TaxValues": [],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": null,
+                            "NetValue": 10,
+                            "TaxValue": 0
+                        }
+                    ]
+                }
+            },
+            "Type": "Payment",
+            "Reason": "PaymentCardMissing",
+            "ExpirationUtc": "2023-02-20T12:00:00Z",
+            "Description": "Payment required",
+            "Notes": "Internal notes."
+        }
+    ],
+    "Cursor": null

```

Comment:
I don't think we should include cursor here, although the result DTO is reused in add + get, cursor will never be filled here. Ideally we should have a different DTO for the addition or update responses, but here I'd consider omitting it.